appium-test/test.js

```diff
@@ -0,0 +1,45 @@
+const {remote} = require('webdriverio');
+
+const capabilities = {
+  platformName: 'Android',
+  'appium:automationName': 'UiAutomator2',
+  'appium:deviceName': 'Android',
+  'appium:appPackage': 'com.auth0.auth0_flutter_example',
+  'appium:appActivity': '.MainActivity',
+  'appium:app': '/home/circleci/project/auth0_flutter/example/build/app/outputs/flutter-apk/app-x86_64-release.apk'
```

Comment:
Any way we can avoid having an absolute path here?

Reply:
Good catch, this is now updated

Reply:
Good catch, this is now updated